app/domain/authentication/authn_jwt/configuration_jwt_generic_vendor.rb

```diff
@@ -12,10 +12,11 @@ def initialize
       end
 
       def create_authentication_parameters(authenticator_input)
-        @authentication_parameters = @authentication_parameters_class.new(authenticator_input)
-        @authentication_parameters.jwt_token = @extract_token_from_credentials.call(
-          credentials: authenticator_input.request.read
+        authentication_parameters = @authentication_parameters_class.new(authenticator_input)
+        authentication_parameters.jwt_token = @extract_token_from_credentials.call(
```

Comment:
can the `jwt_token` be passed to the initializer instead of being an `attr_accessor`?

Reply:
It will make us to change all the unit tests of the authneticator. And the Status API of the authenticator that @Avi-Eyal is doing uses the authentication parameters without jwt token